src/main/java/com/twingly/search/Query.java

```diff
@@ -0,0 +1,157 @@
+package com.twingly.search;
+
+import com.twingly.search.exception.TwinglyException;
+
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * The type Query.
+ */
+public class Query {
+    private static final String BASE_URL = "http://api.twingly.com";
```

Comment:
Unfortunately - no.
If we want to use betamax with https right now (actually, they are going to publish new beta in the beginning of march) - we need to use httpclient also to retrieve data from API.
(It's how I see it now.)